internal/lefthook/run_command.go

```diff
@@ -0,0 +1,40 @@
+package lefthook
+
+import (
+	"bytes"
+	"io"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/creack/pty"
+
+	"github.com/evilmartians/lefthook/internal/log"
+)
+
+type WaitFunc func() error
+
+func RunCommand(runner string, cmdRoot string) (*bytes.Buffer, bool, error) {
+	command := exec.Command("sh", "-c", runner)
+	if cmdRoot != "" {
+		fullPath, _ := filepath.Abs(cmdRoot)
+		command.Dir = fullPath
+	}
+
+	return RunPlainCommand(command)
+}
+
+func RunPlainCommand(command *exec.Cmd) (*bytes.Buffer, bool, error) {
+	ptyOut, err := pty.Start(command)
+	if err != nil {
+		return nil, false, err
+	}
+	defer ptyOut.Close()
+
+	commandOutput := bytes.NewBuffer(make([]byte, 0))
```

Comment:
Not critical, yet `new(bytes.Buffer)` does the same thing as `bytes.NewBuffer(make([]byte, 0))`